### ISSUE TITLE
[Lua Extension] Harmony integration

### ISSUE THREAD

**Issue by rainbowdesign** [javisar/ONI-Modloader#56]
It would be nice to add an integration for Harmony to the lua scripting engine. Including getting __instance, __result, and transpilers.

**Comment by rainbowdesign** [comment]
Could you please atleast comment on why you close my idea.
You agreed to host general mod ideas here.

**Comment by javisar** [comment]
I closed it because it's an already existing mod. Ideas of extensions to your mod should go in you own git/repo/workshop...
This was intented for new non existing mod ideas. Despite that, it seems like the mod ideas has been moved to discord and to the forums:
https://forums.kleientertainment.com/forums/forum/204-oxygen-not-included-mods-and-tools/
I suggest you to post ideas there. I don't think anybody use this github anymore. I'll probably move all to the forum and clean this git up.
